test/verify/check-superuser

```diff
@@ -539,6 +539,21 @@ class TestSuperuserDashboard(testlib.MachineCase):
         b.wait_in_text(".super-channel span", 'result: ')
         self.assertIn('result: uid=0', b.text(".super-channel span"))
 
+        # Logging out and logging back in should give us immediate
+        # superuser on m2 (once we have logged in there).
+        b.relogin()
+        b.wait_visible("#machine-troubleshoot")
+        b.click('#machine-troubleshoot')
+        b.wait_visible('#hosts_setup_server_dialog')
+        b.set_input_text("#login-custom-password", "foobar")
+        b.click('#hosts_setup_server_dialog button:contains("Log in")')
+        b.wait_not_present('#hosts_setup_server_dialog')
+
+        b.enter_page("/playground/test", host="10.111.113.2")
+        b.click(".super-channel button")
+        b.wait_in_text(".super-channel span", 'result: ')
+        self.assertIn('result: uid=0', b.text(".super-channel span"))
```

Comment:
This is fine, but `b.check_superuser_indicator("Administrative access")` is a little more direct.

Reply:
Both would be good, I guess.

Reply:
Done